arch/risc-v/src/common/riscv_pmp.c

```diff
@@ -344,8 +342,8 @@ static void pmp_napot_decode(uintptr_t val, uintptr_t * base,
     }
 
   val &= ~mask;
-  *base = (val << 2);
-  *size = (1 << pot);
+  *size = (1ul << pot);
```

Comment:
let's change to `llu` or even `UINT64_C` will work better.
```suggestion
  *size = UINT64_C(1) << pot;
```
The alternative is
```suggestion
  *size = ((uintptr_t)1 << pot);
```

Reply:
No problem